src/modules/IscsiClientLib.rb

```diff
@@ -252,14 +260,12 @@ def GetOffloadCard
       @offload_card
     end
 
-    def SetOffloadCard(new_card)
-      Builtins.y2milestone("SetOffloadCard:%1 cur:%2", new_card, @offload_card)
-      if new_card != @offload_card
-        @offload_card = new_card
-        CallConfigScript() if new_card != "default"
-      end
+    def selected_iface
+      @iface || "default"
+    end
 
-      nil
+    def iface=(iface)
+      log.info "Selecting the iface: #{iface} cur:#{@iface}"
```

Comment:
A setter that actually sets nothing? This is at least surprising.

Reply:
cool... :see_no_evil: